examples/schema_outputs/outlines-converter-example.py

```diff
@@ -0,0 +1,132 @@
+# ========= Copyright 2023-2024 @ CAMEL-AI.org. All Rights Reserved. =========
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ========= Copyright 2023-2024 @ CAMEL-AI.org. All Rights Reserved. =========
+
+from pydantic import BaseModel
+
+from camel.schemas import OutlinesConverter
+
+# Define the model using OutlinesConverter
+model = OutlinesConverter(
+    model_type="microsoft/Phi-3-mini-4k-instruct", platform="transformers"
+)
+
+######## Regex conversion #########
+
+time_regex_pattern = r"(0?[1-9]|1[0-2]):[0-5]\d\s?(am|pm)?"
+output = model.convert_regex(
+    "The the best time to visit a dentist is at ", time_regex_pattern
+)
+
+print(output)
+'''
+6:00 pm
+'''
+
+
+######## JSON conversion #########
+
+
+# 1. Using a Pydantic model
+class Temperature(BaseModel):
+    location: str
+    date: str
+    temperature: float
+
+
+output = model.convert_json(
+    "Today is 2023-09-01, the temperature in Beijing is 30 degrees.",
+    output_schema=Temperature,
+)
+
+print(output)
+'''
+location='Beijing' date='2023-09-01' temperature=30.0
+'''
```

Comment:
the output format use '=' @Wendong-Fan , not {'location':'Beijing'}

Reply:
Thanks, I split the it into 2 functions, one returns pydantic object, one returns the internal dict object.